ansible/roles/projects/ovn-kubernetes/test/tasks/deploy-test-pods.yml

```diff
@@ -0,0 +1,39 @@
+---
+- name: Create ovn-kubernetes pod {{ pod_name }}
```

Comment:
Can the final pod spec be added to the artifacts directory? or maybe use templates to render pods specs and then copy them to artifacts? WDYT?

Reply:
It is in the final commit